src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+featured_stat_1: 93%
+featured_stat_label_1: Websites load at least one tracker
+featured_stat_2: Nine out of ten
+featured_stat_label_2: Largest domains that set cookies on the biggest percentage of websites are Google-owned.
+featured_stat_3: 44.8%
+featured_stat_label_3: Sites have a privacy policy
 unedited: true
 ---
+
+
+## Introduction
+
+This chapter of the Web Almanac gives an overview of the current state of privacy on the web. This topic has been increasing in popularity recently and has raised awareness on the users’ side. The need for guidelines has been met with various regulations (such as [GDPR](https://gdpr-info.eu/) in Europe,  [LGPD](https://lgpd-brazil.info/) in Brazil, [CCPA](https://leginfo.legislature.ca.gov/faces/codes_displayText.xhtml?division=3.&part=4.&lawCode=CIV&title=1.81.5) in California…), that aim to increase the accountability of data processors and their transparency  towards users. In this chapter, we discuss the prevalence of online tracking with different techniques and the adoption rate of cookie consent banners and privacy policies by websites.
+
+
+## Online Tracking
+Third-party trackers collect user data to build up profiles of the user’s behavior to be monetized for advertising purposes. This raises privacy concerns with users on the web, which resulted in the emergence of various tracking protections. However, as we will see in this section, online tracking is still widely used. Not only does it have a negative impact on privacy, online tracking has a [huge impact on the environment](https://gerrymcgovern.com/calculating-the-pollution-cost-of-website-analytics-part-1/) and avoiding it can lead to [better performance](https://twitter.com/fr3ino/status/1000166112615714816). 
+We examine the prominence of the most common types of third-party tracking, namely by means of third-party cookies and the use of fingerprinting. Online tracking is not limited to just these two techniques, new ones keep arising to circumvent existing countermeasures. 
+
+
+### Third-party trackers
+We use [WhoTracksMe](https://whotracks.me/)’s tracker list to determine the percentage of websites that issue a request to a tracker. As in the following figure, we find that on roughly 93% of websites, at least one tracker is present. 
+
+{{ figure_markup(
+  image="websites_that_load_trackers.png",
+  caption="Websites including at least one tracker",
+  description="This figure shows the percentage of websites that load at least one tracker.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1325818112&format=interactive",
+  sheets_gid="1591448294"
```

Comment:
Do we have the sql for this stat? Can't see it in https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/10_Privacy/

Reply:
Didn't add the queries for online tracking since I was using an external dataset (WhoTracksMe)